### ISSUE TITLE
Allow some formatting for how information will be placed in the Roam graph from Matter

### ISSUE THREAD

**Issue by joshjhall** [getmatterapp/roam-matter#3]
When exporting to my Roam graph, I’d like some control over the formatting. For example, I format highlights coming from Readwise something like `[[>]] <highlight content`. This makes the highlights easy to filter, and uses block quote formatting to ensure I don’t accidentally confuse quoted material for a note I took later on. I appreciate this might be a bit advanced, but people using tools like Roam today are probably going to be fairly advanced. There’s also a lot of experimentation continuing around how best to organize graph data, so allowing some level of customization here facilitates exploration.

Importantly, there may be several items injected into the graph with block references and such between them (something Readies fails at today). This complicates the UI/UX for the formatting modifications, but maybe there's an balance between flexibility and preset options that makes sense. For example, I like how Matter is using a block reference for the note taken on the journal date to reference back to the highlight. This allows multiple notes over a period of time to, hypothetically, be added to the same highlight leveraging block references to tie them altogether in the graph.

In short...
* Ability to change (to some level) the format of content from Matter as injected into Roam
* Ability to create block references and multiple entries in Roam to account for the real-world events of highlight and annotate separately in the graph
* Access to additional metadata to use in formatting the content for Roam (e.g., tags, article title, link, publication date, date added to Matter, other bibliographic information available, etc.).

**Comment by stan-voo** [comment]
second this! Would be awesome to have templating abilities.

**Comment by sweenzor** [comment]
Would love (even simple) templating!

**Comment by alapx** [comment]
+1 would love some formatting control as my input use cases on roam are also varied
